CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
```suggestion
# Contributing

```

Minor, but a MD file should have a normal list of `#`.  L4, it starts straight with 2 `##`.